### PR TITLE
PCSUP Fixes

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/runtime-security/configure/permissions.adoc [hunk 1]
```diff
@@ -81,3 +81,17 @@ The following table shows the required permissions for agentless scanning.
 https://main\--prisma-cloud-docs-website\--hlxsites.hlx.live/en/compute-edition/assets/compute-feature-wise-permissions?sheet=azure-agentless-30-03
 |===
 
+=== Agentless Scanning Resource Group Permissions
```
Comment (rcaballeromx) [inline docs/en/enterprise-edition/content-collections/runtime-security/configure/permissions.adoc:84]: @jenjoe22 Did you also update the PCCE page?
I don't see the file listed.
https://docs.prismacloud.io/en/compute-edition/32/admin-guide/configure/permissions

**Review by rcaballeromx — CHANGES_REQUESTED** [review]
I think the on-prem page wasn't updated.
Was that on purpose?

File: docs/en/enterprise-edition/content-collections/runtime-security/configure/permissions.adoc (hunk 1 above)
Comment (jenjoe22) [reply docs/en/enterprise-edition/content-collections/runtime-security/configure/permissions.adoc:84]: @rcaballeromx Omri confirmed that the update does not need to be made for on-prem versions.

**Comment by jenjoe22** [comment]
> I think the on-prem page wasn't updated. Was that on purpose?

Omri confirmed the changes are not required for on-prem.

**Review by rcaballeromx — APPROVED** [review]

**Pull request merged**